OpenEdXMobile/src/main/java/org/edx/mobile/model/course/CourseDatesBannerInfo.kt

```diff
@@ -0,0 +1,11 @@
+package org.edx.mobile.model.course
+
+import com.google.gson.annotations.SerializedName
+
+
+data class CourseDatesBannerInfo(
+        @SerializedName("missed_deadlines") val missed_deadlines: Boolean = false,
+        @SerializedName("missed_gated_content") val missed_gated_content: Boolean = false,
+        @SerializedName("verified_upgrade_link") val verified_upgrade_link: String = "",
+        @SerializedName("content_type_gating_enabled") val content_type_gating_enabled: Boolean = false
```

Comment:
We are adopting `camelCase` notation for naming a variable/data member, so need to update.